.changelog/2650.internal.1.md

```diff
@@ -0,0 +1,9 @@
+replace Roothash, Scheduler and Registry fields with Consensus accessors
+
+ConsensusAPI.Backend contains accessors for RootHash, Scheduler, and Registry.
```

Comment:
Since Change Log is Markdown, use `` ` ``s when referencing types, variables or file paths.